src/Acme/UserBundle/Repository/UserRepository.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
This file doesn't need to be here, it doesn't look like your User entity is even referencing it.